tests/e2e/parameters/infrastructure.py

```diff
@@ -0,0 +1,103 @@
+from pyclbr import Function
+
+from aws_cdk import CfnOutput
+from aws_cdk import aws_appconfig as appconfig
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_ssm as ssm
+
+from tests.e2e.utils.infrastructure import BaseInfrastructure
+
+
+class ParametersStack(BaseInfrastructure):
+    def create_resources(self):
+        functions = self.create_lambda_functions()
+        self._create_parameter_string(function=functions["ParameterStringHandler"])
+        self._create_app_config(function=functions["ParameterAppconfigFreeformHandler"])
+
+    def _create_parameter_string(self, function: Function):
+        parameter = ssm.StringParameter(
+            self.stack, id="string_parameter", parameter_name="sample_string", string_value="Lambda Powertools"
```

Comment:
We need to split SSM into a separate PR. When you do, we need to ensure the parameter name is unique otherwise it'll fail subsequent tests running in parallel

Reply:
I agree 100% @heitorlessa! I removed it and I will raise a new issue for this.